### PR TITLE
Added SSE for browser communication

### PR DESCRIPTION
**Pull request by ergl**

### CODE REVIEW

File: server/job.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+
+job_completed = False
```
Comment (Roboe) [inline server/job.py]: Do you like it `global`? Job could be a class. It has sense, `Storage` is a class too.

Everything else, great, :+1:

Comment (ergl) [inline server/job.py]: See latest commit, `Job` is now a class too

**Comment by Roboe** [comment]
Great, :+1:

File: js/sse.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    });
+
+
+    // TODO: Add origin attribute?
+    source.addEventListener('jobcomplete', function(e) {
+        var response = JSON.parse(e.data);
+        console.log(response);
+        // Tell the user to get the f*ck out
+        // close the stream, we no longer need it
+        source.close()
```
Comment (angelsanz) [inline js/sse.js]: Semicolon!

Comment (Roboe) [inline js/sse.js]: Right! Hahaha

**Pull request merged**